app/templates/Articles/add.php

```diff
@@ -1,8 +1,6 @@
 <h1>Add Article</h1>
 <?php
     echo $this->Form->create($article);
-    // TODO: user_id Hard code for now. we going to fix it soon.
```

Comment:
I think you've forgotten the other TODO https://github.com/ArielGalvez/tutorial-cakephp/blob/ec63e55b991fff09802857a5680ac118d172362b/app/templates/Articles/edit.php#L4

Please check.

Reply:
removed comment and the input hidden